libcloud/compute/drivers/openstack.py

```diff
@@ -1283,19 +1290,32 @@ def _to_images(self, obj, ex_only_active):
 
     def _to_image(self, api_image):
         server = api_image.get('server', {})
+        updated = api_image.get('updated_at') or api_image['updated']
+        created = api_image.get('created_at') or api_image['created']
+        min_ram = api_image.get('min_ram')
+
+        if min_ram is None:
+            min_ram = api_image.get('minRam')
+        min_disk = api_image.get('min_disk')
```

Comment:
nitpick (you don't need to change that): it's surprising to visually group min_ram and min_disk. I guess I would have added newlines *after* the if statements.